src/main/java/pages/AdaugaVenituri.java

```diff
@@ -0,0 +1,146 @@
+package pages;
+
+import base.BaseTest;
+import base.PageLinksAndText;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.testng.Assert;
+
+public class AdaugaVenituri extends BaseTest {
+
+    @FindBy(xpath = "/html/body/main/div/div/div/div[2]/div/div/div/div[1]/form/div[5]/button")
+    private WebElement authenticationButtonLink;
+
+    @FindBy(xpath = "//a[normalize-space()='Creeaza un cont!']")
+    private WebElement createAccountButtonLink;
+
+    @FindBy(xpath = "//input[@id='email']")
+    private WebElement email;
+
+    @FindBy(xpath = "//input[@id='password']")
+    private WebElement passwordField;
+
+
+    @FindBy(xpath = "//label[@for='remember-me']")
+    private WebElement rememberMeCheckbox;
+    @FindBy(xpath = "//button[@class='onetrust-close-btn-handler onetrust-close-btn-ui banner-close-button ot-close-icon']")
+    private WebElement inchideCookie;
+    @FindBy(xpath = "'onetrust-close-btn-handler onetrust-close-btn-ui banner-close-button ot-close-icon']")
+    private WebElement closeCookieButton;
+    @FindBy(xpath = "//button[normalize-space()='Autentificare']")
+    private WebElement loginButton;
+    @FindBy(xpath = "//button[normalize-space()='Autentificare']")
+    private WebElement authenticationButton;
+    @FindBy(xpath = "//a[normalize-space()='ContApp PS']")
+    private WebElement contAppPs;
+    @FindBy(xpath = "//h3[@class='header-title']")
+    private WebElement configurare;
+    @FindBy(xpath = "//h3[@class='header-title']")
+    private WebElement venituri;
+    @FindBy(xpath = "//div[normalize-space()='Articole venituri']")
+    private WebElement articoleVenituri;
+    @FindBy(xpath = "//span[@class='d-none d-sm-inline-block']")
+    private WebElement adaugaArticolNou;
+    @FindBy(xpath = "//textarea[@id='textarea-name']")
+    private WebElement denumireArticol;
+    @FindBy(xpath = "//span[@id='select2-select-select-currency-container']")
+    private WebElement valuta;
+    @FindBy(xpath = "//input[@id='input-sellingPrice']")
+    private WebElement pretVanzare;
+
+    @FindBy(xpath = "//span[@id='select2-select-select-articleTypeId-container']")
+    private WebElement tipArticol;
+
+    @FindBy(xpath = "//span[@id='select2-select-select-categoryId-container']")
+    private WebElement categorieFiscala;
+
+    @FindBy(xpath = "//span[@id='select2-select-select-measurementUnitId-container']")
+    private WebElement unitateDeMasura;
+    @FindBy(xpath = "//button[@class='btn btn-primary']")
+    private WebElement salveaza;
+
+    public void clickCloseCookie() {
+        try {
+            click(closeCookieButton);
+        } catch (Exception e) {
+            System.out.println("Cookie is not displayed");
```

Comment:
nu folosim Sout in code. Te rog sa folosesti Assert

Reply:
Am rezolvat